src/data/list/zip.lean

```diff
@@ -273,4 +273,14 @@ begin
     { simp [hl] } }
 end
 
+@[simp] lemma sum_zip_with_distrib_left (f : α → β → ℕ) (n : ℕ) (l : list α) (l' : list β) :
```

Comment:
Could we generalize from `nat` to any semring? (Perhaps even to any semimodule if viable with the imports.)

Reply:
Only semiring is available in the imports, not semimodule.